CHANGES.md

```diff
@@ -1,6 +1,6 @@
-# Unreleased
+# 2.8.3 (2022-04-04)
```

Comment:
Let's bump this to tomorrow to give time to run the large regression test suite before doing the release
```suggestion
# 2.8.3 (2022-04-05)
```

Reply:
Rebased in 8620476